draco/core/containers.py

```diff
@@ -1252,6 +1252,34 @@ def _mean(self):
         return self.datasets["vis"]
 
 
+class SiderealStreamRebin(SiderealStream):
```

Comment:
This class does not define `vis`, `weight`, `input_flag`, etc., datasets in the `_dataset_spec` dictionary.  I am finding that if I attempt to load a `SiderealStreamRebin` from disk with selections along an axis, then the selection will only be applied to the datasets that are currently defined in `_dataset_spec`.  I think we need to either:
* include all possible datasets in `_dataset_spec`
* have `_dataset_spec` populated from subclasses, similar to what is done with the `_axes` attribute
* update the code that does the axis selection to get the lists of datasets from some other source (and update anywhere else in the code that uses this attribute)

Reply:
Looks like this is fixed in #260, I've updated that PR